### ISSUE TITLE
index predicates not supported

### ISSUE THREAD

**Issue by JerrySievert** [oguimbal/pgsql-ast-parser#52]
hi there!

first off, I love the project, pretty much anything with javascript and Postgres make me very happy.

I've been looking at writing some static analysis code, and ran into an issue where index predicates cause the parser to fail:

`CREATE INDEX test ON table_name USING btree (column_name) WHERE column_name IS NOT NULL`

returns:

```
Error: Syntax error at line 1 col 59:

  CREATE INDEX test ON table_name USING btree (column_name) WHERE
                                                            ^
Unexpected kw_where token: "where". Instead, I was expecting to see one of the following:

    - A "semicolon" token
```

**Comment by oguimbal** [comment]
Hi,

Sorry for the delay, I dont have enough time to work on this these days.

I'll try to have a look when I have time.

Thanks for the report, and for the kind words

**Comment by oguimbal** [comment]
Hi ! Fixed in `pgsql-ast-parser@9.1.0`. Sorry for the delay :)

**Comment by JerrySievert** [comment]
finally able to say thank you! it is much appreciated!
